scripts/fix_style.py

```diff
@@ -17,11 +17,19 @@ def recursive_file_list(path):
 	"src/engine/client/keynames.h",
 	"src/engine/keys.h",
 ]
+IGNORE_DIRS = [
+	"src/game/generated",
```

Comment:
Actually I think these need `/` at the end to be correct, otherwise it would match `src/game/generated.cpp` too.
```suggestion
	"src/game/generated/",
```

Reply:
Sadly adding `/` at the end and then calling `realpath` removes the trailing path separator. Instead, I added `os.path.sep` in the check